apps/condo/domains/ticket/schema/TicketShareService.js

```diff
@@ -0,0 +1,56 @@
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const { COUNTRIES, RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { SHARE_TICKET_MESSAGE_TYPE } = require('@condo/domains/notification/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+const access = require('@condo/domains/ticket/access/TicketReportService')
+
+const TicketShareService = new GQLCustomSchema('TicketShareService', {
+    types: [
+        {
+            access: true,
+            type: 'input TicketShareInput { sender: JSON!,  users: [ID!]!, ticketId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type TicketShareOutput { status: String! }',
+        },
+    ],
+    mutations: [
+        {
+            access: access.canReadTicketReportWidgetData,
```

Comment:
! you should create own access check

Reply:
It seems logical that anyone with read permission can share the ticket, right? 
(now: `access.canReadTickets`)

Reply:
We have folder access and in this folder, you can add TicketShareService.js with exported function
"canShareTicket" and, for now, add "canReadTickets" inside this check 
Later we can manage such permissions if we want to allow only some roles  to share tickets (for example, ticket's operators can share tickets, but not all members of the organization)